### ISSUE TITLE
Memory Test for phet brand

### ISSUE THREAD

**Issue by SaurabhTotey** [phetsims/number-line-distance#36]
Standard code review issue. Created for #35. I'll perform the memory test and put the data here soon.

**Comment by SaurabhTotey** [comment]
First run: Tested using the built_all_debug version from master, SHA was 3a7c86f731fb68ffa3882c10b835ce8c75969055, common code was whatever was on master today, 07/13/2021.

Test | Memory Usage (MB)
--- | --- 
Immediately after load, no fuzzing | 22.1
Load, visit each screen, then back to home screen, no fuzzing | 22.6
After 1 min of fuzz | 27.8
After 2 min of fuzz | 28.6
After 3 min of fuzz | 28.6
After 4 min of fuzz | 28.9
After 5 min of fuzz | 29.2
Stop fuzz, reset each screen, go to home screen | 29.3

This seems like it is a reasonable memory growth profile. With @jbphet, we compared the post-fuzz and pre-fuzz snapshots and saw no leakage of sim-specific classes. I will leave this issue open, and let @pixelzoom close at their discretion when doing the code review.
